Bottom: 1d4cff46b27d0f6192559ee1fed0c2e7fc192320
Top:    6f80dc7afbe5aaa0bc8a6669b5128ded4882aa25
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 14:32:20 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 3bfd354..0018b9d 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -888,13 +888,6 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
   glClear (GL_COLOR_BUFFER_BIT | GL_STENCIL_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
   hidgl_reset_stencil_usage ();
 
-#if 0
-  glClearColor (gport->bg_color.red / 65535.,
-                gport->bg_color.green / 65535.,
-                gport->bg_color.blue / 65535.,
-                1.);
-#endif
-
 #if 1
   region.X1 = MIN (Px (ev->area.x), Px (ev->area.x + ev->area.width + 1));
   region.X2 = MAX (Px (ev->area.x), Px (ev->area.x + ev->area.width + 1));
